security/libkmip-tests.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright: 2021 IBM
+# Author: Nageswara R Sastry <rnsastry@linux.vnet.ibm.com>
+
+import os
+from avocado import Test
+from avocado.utils import archive, build, distro, process
+from avocado.utils.software_manager import SoftwareManager
+
+
+class libkmip(Test):
+    """
+    libkmip-testsuite
+    :avocado: tags=security,testsuite
+    """
+    def setUp(self):
+        '''
+        Install the basic packages to support libkmip
+        '''
+        # Check for basic utilities
+        smm = SoftwareManager()
+        deps = ['gcc', 'make', 'autoconf']
+        self.distro_name = distro.detect().name
+        for package in deps:
```

Comment:
if we not changing package list so feel we can avoid to have extra variable .

can be like 
```
for package in ['gcc', 'make', 'autoconf']:
```